radicle-node/src/main.rs

```diff
@@ -34,14 +36,22 @@ impl Options {
                     let addr = parser.value()?.parse()?;
                     listen.push(addr);
                 }
+                Long("external-address") => {
```

Comment:
bitcoind uses '-externalip'

Reply:
ok yes this makes sense, how do they specify the port?

Reply:
I recommend to keep ports.  There is an important distinction where radicle serves user information in a way bitcoin does not.  Bitcoin's information is held on a unified ledger vs. projects sparsely hosted across the network.  Prioritizing connectivity is more important for us (multiple users on a local area network etc.).